go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/Khan/genqlient v0.4.1-0.20220330204801-b2422452a143
 	github.com/google/go-github/v42 v42.0.0
 	github.com/hashicorp/vault/api v1.4.1
-	github.com/keybase/go-crypto v0.0.0-20200123153347-de78d2cb44f4
+	github.com/keybase/go-crypto v0.0.0-20161004153544-93f5b35093ba
```

Comment:
Kind of. The whole purpose for writing this in golang was to mimic the encryption code of the terraform provider. And the version I downgrade to here is the version the provider uses. This implicitly breaks usage of ECC :see_no_evil:

Reply:
ok 😅 

Can you add a comment in this file explaining this? So that it is clearer that this has to match whatever terraform has. I guess tests will fail too, so everything will be easier to understand.